src/lib/lessons.ts

```diff
@@ -156,22 +156,6 @@ const loadLessonGraphQLQuery = /* GraphQL */ `
         slug
         twitter
       }
-      comments {
```

Comment:
Comment loading happens on these lines and is then merged in with the results of this graphql request.

https://github.com/eggheadio/egghead-next/blob/main/src/lib/lessons.ts#L77-L81

<img width="663" alt="CleanShot 2022-08-02 at 09 54 25@2x" src="https://user-images.githubusercontent.com/694063/182405316-3fe6dd73-123d-465d-8014-31d0987874db.png">